docs/user-guide/configure.md

```diff
@@ -178,10 +187,11 @@ You can use the `url` secondary option to provide a custom link to external docs
 
 For example:
 
-<!-- prettier-ignore -->
 ```json
 {
-  "color-no-hex": [true, {"url": "https://example.org/your-custom-doc"}]
+  "rules": {
+    "color-no-hex": [true, { "url": "https://example.org/your-custom-doc" }]
```

Comment:
nit

replace `color-no-hex` by `namespace/foo-qux`

Reply:
This case is to override the default URL for the built-in `color-no-hex`, so the current example is fine to me.